locale/po/datastore_search.po

```diff
@@ -333,78 +246,70 @@ msgid ""
 "            >\n"
 "        "
```

Comment:
This may be just how PO files are, but it seems weird to have spaces between the quotes `"` ?

Reply:
Yeah, I think it's just the PO file implementation. It's quoting the indentation from the `.js` file. No harm in it, though.